### ISSUE TITLE
Links in Hipchat messages don't work

### ISSUE THREAD

**Issue by cici** [jlewallen/jenkins-hipchat-plugin#49]
At the end of my Hipchat message, there is a link titled (Open).  Nothing happens when I click on it.  I would assume it would take me to the corresponding Jenkins job or be configurable.

**Comment by sanbornc** [comment]
Have you set the Project Room Id?

**Comment by cici** [comment]
Yes, the messages show up in the proper Hipchat room, but the link back to Jenkins doesn't do anything.  It is not using the value I have in Build Server URL as the base of the URL.

**Comment by sanbornc** [comment]
Sorry... was not paying attention is the "Jenkins URL" set in "Global HipChat Notifier Settings"

**Comment by cici** [comment]
Yes, it is called "Build Server URL" and it is not being included in the URL on the Open link.  The URL does end in a /

**Comment by sanbornc** [comment]
This pull request by Matt Ferrante may be the fix you are looking for

https://github.com/jlewallen/jenkins-hipchat-plugin/pull/48

**Comment by cici** [comment]
I think you are correct. Do you know when the next release is going to be?

Thanks!

Charlie Sanborn wrote:

**Comment by ferrants** [comment]
Ah! I ran into this, so I fixed it on my branch and put in the pull request. It doesn't look like there has been a release in a long time. Installing via Jenkins pulls a version that is two years old, I think.

@cici - are you building this and manually installing it? 

Edit: meant to reference cici and not sanbornc

**Comment by cici** [comment]
No, I installed using the Manage Plugins feature of Jenkins.

Matt Ferrante wrote:

**Comment by ferrants** [comment]
Which version does it say you have installed?

**Comment by cici** [comment]
0.1.5-SNAPSHOT (private-03/31/2013 20:35-marcin.szczepanski)

Matt Ferrante wrote:

**Comment by ferrants** [comment]
It doesn't look like you installed by selecting from the /pluginManager/available page, but downloaded the built plugin from somewhere and installed by uploading it. 0.1.4 is the version available through the auto-installer and it doesn't have this bug. It also doesn't have some of the features of 0.1.5, like only sending messages on failures.

If you downloaded the plugin from somewhere, you will probably have to download and install the plugin again to update the version. The plugin in the Jenkins repository hasn't been updated in a while, probably because of bugs with the source code in the repo's master branch.

**Comment by cici** [comment]
Been a while so I don't really remember.  I will try installing it manually from that pull request.

Thanks!

**Comment by cici** [comment]
That fixed the problem.

Thanks for your help!

Matt Ferrante wrote:
